python/foglamp/plugins/south/http_south/http_south.py

```diff
@@ -71,13 +71,14 @@ def plugin_init(config):
     handle = config
     return handle
 
-def plugin_start(data):
+def plugin_start(data, loop=None):
```

Comment:
This entry point is specified in the API document as taking only data as input. We should not change this.